addons/mail/static/src/components/message/message.xml

```diff
@@ -91,7 +91,7 @@
                             <t t-if="message.moderation_status === 'pending_moderation' and !message.isModeratedByCurrentPartner">
                                 <span class="o_Message_moderationPending o-author" title="Your message is pending moderation.">Pending moderation</span>
                             </t>
-                            <t t-if="message.notifications.length > 0">
+                            <t t-if="!hasDifferentOriginThread and message.notifications.length > 0">
```

Comment:
Too bad double negation. Think would be easier to read if we change getter to output the inverse (& change name ofc).

Reply:
"hasSameOriginThread" ?

Reply:
brainstorming: `isOriginatingFromCurrentThread`

Reply:
> brainstorming: `isOriginatingFromCurrentThread`

Yes, best naming so far.